plugin.go

```diff
@@ -382,6 +323,90 @@ func (p *Plugin) meta(environment, tags string, options metaOptions) error {
 			return errJoin
 		}
 
+	} else {
+		launchr.Term().Info().Println("Starting CI build (now default behavior)")
+
+		gitlabDomain := "https://projects.skilld.cloud"
+		launchr.Term().Info().Printfln("Getting %s credentials from keyring", gitlabDomain)
+		ci, save, err := getCredentials(gitlabDomain, username, password, p.k)
+		if err != nil {
+			return err
+		}
+		launchr.Term().Printfln("URL: %s", ci.URL)
+		launchr.Term().Printfln("Username: %s", ci.Username)
+
+		username := ci.Username
+		password := ci.Password
+
+		comparisonRef := ""
+		if options.override != "" {
```

Comment:
now that started thinking about it, would be great to copy truncate function from bump to meta, so we always have 7 symbols for override.